tox.ini

```diff
@@ -37,10 +38,12 @@ deps = -r{toxinidir}/requirements-tests.txt
 [testenv:py2.6-lxml]
 deps = -r{toxinidir}/requirements-tests.txt
        unittest2
+       mock
```

Comment:
mock is already specified in requirements-tests.txt so no need to also specify it here.

In any case, not a big deal, I will remove it while merging :)